src/EnergyPlus/PythonLibWrapper.cc

```diff
@@ -85,6 +86,8 @@ extern "C" {
     PYTHONWRAP_API PyObject * EP_Wrap_PyObject_CallObject(PyObject *callable, PyObject *args) {return PyObject_CallObject(callable, args);}
     PYTHONWRAP_API PyObject * EP_Wrap_PyObject_GetAttrString(PyObject *o, const char *attr_name) {return PyObject_GetAttrString(o, attr_name);}
     PYTHONWRAP_API PyObject * EP_Wrap_PyObject_CallFunction(PyObject *callable, const char *format) {return PyObject_CallFunction(callable, format);}
+    PYTHONWRAP_API PyObject * EP_Wrap_PyObject_CallFunction3Args(PyObject *callable, const char *format, PyObject * arg1, PyObject * arg2, PyObject * arg3) {return PyObject_CallFunction(callable, format, arg1, arg2, arg3);}
+
```

Comment:
I add to register this one so I could call a function with 3 args and a format string.



`PyObject_CallFunction` is actually variadic, something the wrapping is losing. I couldn't really figure out how to forward these variable number of args, so I just added a wrap for my specific number of args that I needed.

Note the `...` in the function signature:

`PyObject* PyObject_CallFunction(PyObject *callable, const char *format, ...)`

https://docs.python.org/3/c-api/object.html#c.PyObject_CallFunction

Reply:
Understood, I don't imagine we'll need any additional instances of CallFunction, so this is a suitable addition.